core/controllers/editor_test.py

```diff
@@ -743,15 +744,15 @@ def test_state_yaml_handler_with_no_state_dict_raises_error(self):
 
         self.post_json(
             '/createhandler/state_yaml/%s' % exp_id, {},
-            csrf_token=csrf_token, expected_status_int=404)
+            csrf_token=csrf_token, expected_status_int=400)
 
         self.logout()
 
     def test_exploration_download_handler_with_invalid_exploration_id(self):
         self.login(self.OWNER_EMAIL)
 
         self.get_json(
-            '/createhandler/download/invalid_exploration_id',
```

Comment:
exploration_id must be within the range in order to check the correct behaviour of the test case.
Similar for other cases.